packages/wallet/api/src/lib-wallet.js

```diff
@@ -786,6 +802,7 @@ export function makeWallet({
       : brandTable.initIssuer(issuer, addMeta);
     const { brand } = await recP;
     await initIssuerToBoardId(issuer);
+    await initBrandToBoardId(brand);
```

Comment:
do these always get init-ed together? That would be a helpful constraint. You wouldn't have to check each map before inserting, just either one.

Consider adding `brand` as an arg to `initIssuerToBoardId(issuer, brand)`. Or if there are cases that issuer has to be inited without brand making a new function.

Reply:
Yea I think that's fine, done.